package.json

```diff
@@ -41,7 +41,7 @@
   "engineStrict": true,
   "browserslist": "defaults, not ie 11",
   "engines": {
-    "node": ">=14.15.0"
+    "node": ">=18.17.1"
```

Comment:
nice

Reply:
Would this be too aggressive? It may be loosen to `>=18.x.0` while I am not sure which x is proper.

Reply:
> Would this be too aggressive? It may be loosen to `>=18.x.0` while I am not sure which x is proper.

good idea, engines support using ">=18"